pom.xml

```diff
@@ -21,6 +21,12 @@
             <artifactId>spring-boot-starter-web</artifactId>
             <version>1.3.3.RELEASE</version>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.16.8</version>
```

Comment:
Please move version to properties section.
Additionally check who your IDE support lombok while debugging.